README.md

```diff
@@ -19,7 +19,7 @@ $ "$GOPATH/bin/wuzz" --help
 ### Commands
 
 Keybinding                              | Description
-----------------------------------------|------------------------------------------------------------
+----------------------------------------|---------------------------------------
```

Comment:
Perhaps this is too nitpicky, but this change doesn't seem to fit with this commit or PR.